packages/blitz-rpc/src/data-client/rpc.ts

```diff
@@ -93,18 +96,25 @@ export function __internal_buildRpcClient({
       serialized = serialize(params)
     }
 
+    const body = JSON.stringify({
+      params: serialized.json,
+      meta: {
+        params: serialized.meta,
+      },
+    })
```

Comment:
What the heck, I thought I left a comment here but I don't see it anymore.

Let's move this JSON.stringify call down inline with `body: httpMethod === "POST" ? body : undefined,` so that json.stringify only runs for method POST.